### PR TITLE
[Issue-54] Add TLS support

### PR DESCRIPTION
**Pull request by gaol**
Motivation:

Fixes: #54 

Conformance:

Your commits should be signed and you should have signed the Eclipse Contributor Agreement as explained in https://github.com/eclipse/vert.x/blob/master/CONTRIBUTING.md
Please also make sure you adhere to the code style guidelines: https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines

This PR tries to add TLS support to the python client.
It also updates the README on the options and how to publish.

### CODE REVIEW

**Comment by gaol** [comment]
@BFergerson would you please evaluate this? I will ask for a release once this gets done.

File: python/README.md [hunk 1]
```diff
@@ -49,3 +100,23 @@ pip install dist/vertx-eventbus-client*.tar.gz
...
 
+## Build and Deploy to PyPi:
+
+* Update the version in the `setup.py`:
+```shell
+# change this value to the desired version, like: 1.0.0
+version="1.0.0.dev0"
+```
+
+* Use the following command to build and public to PyPi:
```
Comment (BFergerson) [inline python/README.md:111]:
```suggestion
* Use the following command to build and publish to PyPi:
```

**Review by BFergerson — APPROVED** [review]
I wasn't expecting this so thanks again. Couldn't find any issues :+1:

File: python/README.md (hunk 1 above)
Comment (gaol) [reply python/README.md]: done

**Pull request merged**